Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\loader_tags.py", line 208, in render
    return template.render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 177, in render
    return self._render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\templatetags\static.py", line 116, in render
    url = self.url(context)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\templatetags\static.py", line 113, in url
    return self.handle_simple(path)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\templatetags\static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\contrib\staticfiles\storage.py", line 173, in url
    return self._url(self.stored_name, name, force)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\contrib\staticfiles\storage.py", line 152, in _url
    hashed_name = hashed_name_func(*args)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\contrib\staticfiles\storage.py", line 472, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'tours/style.css'
Not Found: /js/jquery.js
Not Found: /destination/8/js/jquery.js
Not Found: /images/wendy-winarno-timeiBVLB8E-unsplash.jpg
Not Found: /images/Candi_Hindu_Prambanan_BtPlKb7.jpg
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /favicon.ico
Not Found: /js/jquery.js
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\views\generic\base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\views\generic\list.py", line 154, in get
    self.object_list = self.get_queryset()
  File "C:\Users\richo\PycharmProjects\tourmate\tours\views.py", line 50, in get_queryset
    user_city = self.get_ip()
  File "C:\Users\richo\PycharmProjects\tourmate\tours\views.py", line 26, in get_ip
    city = g.city(ip)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\contrib\gis\geoip2\base.py", line 181, in city
    return City(self._city.city(enc_query))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\geoip2\database.py", line 150, in city
    return cast(City, self._model_for(geoip2.models.City, "City", ip_address))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\geoip2\database.py", line 253, in _model_for
    (record, prefix_len) = self._get(types, ip_address)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\geoip2\database.py", line 240, in _get
    raise geoip2.errors.AddressNotFoundError(
geoip2.errors.AddressNotFoundError: The address 127.0.0.1 is not in the database.
Not Found: /favicon.ico
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\views\generic\base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\views\generic\list.py", line 154, in get
    self.object_list = self.get_queryset()
  File "C:\Users\richo\PycharmProjects\tourmate\tours\views.py", line 50, in get_queryset
    user_city = self.get_ip()
  File "C:\Users\richo\PycharmProjects\tourmate\tours\views.py", line 26, in get_ip
    city = g.city(ip)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\django\contrib\gis\geoip2\base.py", line 181, in city
    return City(self._city.city(enc_query))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\geoip2\database.py", line 150, in city
    return cast(City, self._model_for(geoip2.models.City, "City", ip_address))
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\geoip2\database.py", line 253, in _model_for
    (record, prefix_len) = self._get(types, ip_address)
  File "C:\Users\richo\PycharmProjects\tourmate\venv\lib\site-packages\geoip2\database.py", line 240, in _get
    raise geoip2.errors.AddressNotFoundError(
geoip2.errors.AddressNotFoundError: The address 127.0.0.1 is not in the database.
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /favicon.ico
Not Found: /js/jquery.js
Not Found: /favicon.ico
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /favicon.ico
Not Found: /js/jquery.js
Not Found: /favicon.ico
Not Found: /js/jquery.js
Not Found: /images/wisata-candi-prambanan-1-l.jpg
Not Found: /favicon.ico
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
Not Found: /js/jquery.js
